### DISCUSSION TITLE
Support for quick edit popup like Cursor's ctrl+k

### DISCUSSION THREAD

**Discussion by brandones** [glowingjade/obsidian-smart-composer#175] [Ideas & Feature Requests]
It would be nice to be able to select some content, press a hotkey, and type some instructions for modifying that content, for which the diff is automatically brought up. This is how Cursor's ctrl+k works and it is a beautiful feature.

![image](https://github.com/user-attachments/assets/2284fa1c-8481-460e-86ee-6338ee8cb417)

**Comment by glowingjade** [comment]
Thanks for the suggestion – this is definitely on our roadmap!

**Reply by mu-hashmi** [reply]
Is this feature currently being worked on? If not, would love to do so.
